daft/dataframe/dataframe.py

```diff
@@ -138,27 +139,30 @@ def explain(self, show_all: bool = False, simple: bool = False) -> None:
                 unoptimized logical plan.
             simple (bool): Whether to only show the type of op for each node in the plan, rather than showing details
                 of how each op is configured.
+            file (Optional[io.IOBase]): Location to print the output to, or defaults to None which defaults to the default location for
+                print (in Python, that should be sys.stderr)
```

Comment:
default is `sys.stdout` (https://docs.python.org/3/library/functions.html#print)

Reply:
TIL!